examples/euler_3d/test_io.py

```diff
@@ -0,0 +1,61 @@
+import os
```

Comment:
I haven't worried about where to put it, since nose picks up everything.  Currently we run nose from the examples directory, though.